edgelet/edgelet-core/Cargo.toml

```diff
@@ -18,7 +18,7 @@ num_cpus = "1.8.0"
 serde = "1"
 serde_json = "1"
 thiserror = "1"
-tokio = { version = "1", features = ["sync"] }
+tokio = { version = "1", features = ["parking_lot", "sync"] }
```

Comment:
`parking_lot` is not strictly necessary here since the crate does not use `tokio::sync::Mutex`. However, keeping it here would prevent accidental use of `tokio::sync::Mutex` without `parking_lot`.